### PR TITLE
cargo-nextest 0.9.26 (new formula)

### PR DESCRIPTION
**Pull request by rouge8**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/cargo-nextest.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  homepage "https://nexte.st"
+  url "https://github.com/nextest-rs/nextest/archive/refs/tags/cargo-nextest-0.9.26.tar.gz"
+  sha256 "2d06259263e01cbde94efbe1aa489889143539de5e5f84141dac833a4ad0e89a"
+  license "Apache-2.0"
+
+  depends_on "rust" # uses `cargo` at runtime
+
+  def install
+    system "cargo", "install", "--no-default-features", "--features", "default-no-update",
+      *std_cargo_args(path: "cargo-nextest")
```
Comment (carlocab) [inline Formula/cargo-nextest.rb:12]:
```suggestion
    system "cargo", "install", "--no-default-features", "--features", "default-no-update",
                    *std_cargo_args(path: "cargo-nextest")
```

**Review by carlocab — COMMENTED** [review]
Just one small suggestion.

**Review by carlocab — APPROVED** [review]
Thanks!

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@rouge8, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**